src/module/system/settings/homebrew/helpers.ts

```diff
@@ -23,7 +23,7 @@ function isHomebrewCustomDamage(value: object): value is Record<string, CustomDa
         (value) =>
             isObject<CustomDamageData>(value) &&
             typeof value.label === "string" &&
-            ["physical", "energy"].includes(value.category ?? ""),
+            ["physical", "energy", null].includes(value.category ?? ""),
```

Comment:
```suggestion
            ["physical", "energy", ""].includes(value.category ?? ""),
```

Reply:
The type we are doing a typeguard for doesnt allow "" there nor undefined, only null. The way i did it makes it a required property.

?? "" is because typescript complains, but we could do tupleHasValue

Reply:
`value.category ?? ""` is never `null`

Reply:
true. I opened it up to allow undefined as well.